python/lsst/analysis/ap/nb_utils.py

```diff
@@ -70,3 +80,203 @@ def make_simbad_link(ra, dec, radius_arcsec=3.0):
         print(f"No matched sources within {radius_arcsec} arcseconds.")
 
         return None
+
+
+def compareDB(repo1, repo2, coll1, coll2, query1, query2,
+              diffimg_type1='goodSeeingDiff', diffimg_type2='goodSeeingDiff',
+              badFlagList=None, matchRadius=0.1, displayCutouts=False,
+              cutoutPath=None, chunk_size=10000,
+              add_cutout_metadata=True, njobs=1):
+    """ compare two APDB datasets by extracting unassociated sources,
+    spatially crossmatching, and plotting cutouts of the differences.
+
+    Parameters
+    ----------
+
+    repo1: `str`
+        Path to `lsst.daf.butler` repo containing the first dataset.
```

Comment:
It's not really a path, it's a URI or an alias. In some cases it might be a path but for `/repo/embargo` that's not a path at all, it's an alias that turns into an S3 URI.

Maybe "Location of butler repo" ?

Reply:
See my comment above: we should just be passing initialized butlers, not paths, to utility functions like this.

Reply:
Ah, missed these comments before I made my recent push to fix some linter errors.

Reply:
Now passing in pre-initialized Butlers.